README.md

```diff
@@ -172,6 +181,52 @@ options.metrics.histogram('my_histogram', 42, ['foo', 'bar']);
 All metrics are automatically prefixed by the config-provided service name /
 graphite hierachy prefix to ensure a consistent graphite metric hierarchy.
 
+# Rate limiting
+
+Service-runner provides an efficient ratelimiter instance backed by
+[limitation](https://github.com/gwicke/limitation). All per-request checks are
+done in-memory for low latency and minimal overhead.
+
+To enforce a limit:
+```javascript
+// Sets limit to 10 req/s, returns true if above limit. 
+var isAboveLimit = options.ratelimiter.isAboveLimit('some_limit_key', 10);
+```
+
+Several backends are supported. By default, a simple in-memory backend is
+used. For clusters, a [Kademlia DHT](https://en.wikipedia.org/wiki/Kademlia)
+based backend is available. Basic Kademlia configuration:
+
+```yaml
+ratelimiter:
+  type: kademlia
+  # Cluster nodes
+  seeds:
+    # Port 3050 used by default
+    - 192.168.88.99
+```
+
+Advanced Kademlia options:
+```yaml
+ratelimiter:
+  type: kademlia
+  # Cluster nodes
+  seeds:
+    # Port 3050 used by default
+    - 192.168.88.99
+    - address: some.host.com
+      port: 6030
+
+  # Optional
+  # Address / port to listen on
+  # Default: localhost:3050, random port fall-back if port used
+  listen:
+    address: localhost
+    port: 3050
+  # Counter update / block interval; Default: 10000ms
+  interval: 10000
+```
+
```

Comment:
Could be nice to also add an API reference, e.g. describe the `isAboveLimit` method.